version/version.go

```diff
@@ -9,3 +9,12 @@ package version // import "go.mongodb.org/mongo-driver/version"
 
 // Driver is the current version of the driver.
 var Driver = "v1.11.0-prerelease"
+
+// MinSupportedWire is the minimum wire version supported by the driver.
+var MinSupportedWire int32 = 6
```

Comment:
When experimenting with logic between the topology package and the description package, I noticed that the description package is a dependency of the topology package. However, the topology package is where we define the version range making it a requirement to hard-code version data in the description pacakge. I think it makes sense to decouple the version data into the version package to be importable anywhere.